instant_ngp_3dml/rendering.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+"""Rendering Script"""
+import os
+from typing import Dict, Tuple
+
+import imageio
+import numpy as np
+import pyngp as ngp  # noqa
+from tqdm import tqdm
+
+from instant_ngp_3dml import logger
+from instant_ngp_3dml.utils.tonemapper import linear_to_srgb
+from instant_ngp_3dml.utils.tonemapper import tonemap
+
+from utils_3dml.file.json_utils import read_json
+from utils_3dml.monitoring.profiler import profile
+
+
+RENDER_MODES: Dict[str, ngp.RenderMode] = {"depth": ngp.RenderMode.Depth,
+                                           "color": ngp.RenderMode.Shade,
+                                           "confidence": ngp.RenderMode.Confidence}
+
+
+@profile
+def __save_color(outname, image):
+    image = np.copy(image)
+    # Unmultiply alpha
+    image[..., 0:3] = np.divide(
+        image[..., 0:3],
+        image[..., 3:4],
+        out=np.zeros_like(image[..., 0:3]),
+        where=image[..., 3:4] != 0)
+    image[..., 0:3] = linear_to_srgb(image[..., 0:3])
+    image = (np.clip(image, 0.0, 1.0) * 255.0 + 0.5).astype(np.uint8)
+
+    # Some NeRF datasets lack the .png suffix in the dataset metadata
+    if os.path.splitext(outname)[1] != ".png":
+        outname = outname + ".png"
```

Comment:
Pour eviter le *.jpg.png
```suggestion
        outname = os.path.splitext(outname)[0] + ".png"
```